common/src/main/java/org/opensearch/ml/common/model/MLDeploySetting.java

```diff
@@ -25,37 +25,50 @@
 @Getter
 public class MLDeploySetting implements ToXContentObject, Writeable {
     public static final String IS_AUTO_DEPLOY_ENABLED_FIELD = "is_auto_deploy_enabled";
-    public static final String MODEL_TTL_FIELD = "model_ttl";
+    public static final String MODEL_TTL_HOURS_FIELD = "model_ttl_hours";
+    public static final String MODEL_TTL_MINUTES_FIELD = "model_ttl_minutes";
```

Comment:
Why separate hours and minutes? Seems use minutes should be enough.

Reply:
Initially only used Hours. But it may take too much time to test expiration in hours. Having both hours and minutes cover most cases. Most likely only hours will be used, e.g. 72 hours (6 days), etc.

Reply:
Why not just use minutes ?  24 hours will be 24 * 60,

Reply:
It should be easier to set a longer time in hours? For 1 week of 7 days, it would be 10080 minutes, but this is not an obvious number to understand the time length.

Reply:
I would suggest just use the minimal time unit for this, that could be easier. If you add hours, why not add days and months?

Reply:
Done.